venmo-sdk/Venmo.m

```diff
@@ -298,6 +298,22 @@ - (void)validateAPIRequestWithCompletionHandler:(VENGenericRequestCompletionHand
 }
 
 
+#pragma mark - Friends
+
+- (void) getFriendsWithLimit:(NSNumber *)limit beforeUserID:(NSString *)beforeUserID afterUserID:(NSString *)afterUserID completionHandler:(VENGenericRequestCompletionHandler)handler {
+    
+    [self validateAPIRequestWithCompletionHandler:handler];
+    
+    NSMutableDictionary* parameters = [NSMutableDictionary dictionaryWithDictionary:@{@"access_token": self.session.accessToken, @"before": beforeUserID, @"after": afterUserID, @"limit": limit}];
```

Comment:
This line will crash if the user does not supply `limit`, `beforeUserID`, or `afterUserID`.

Let's add them by checking if they are nil first. Like so:

```objc
if (limit) {
    parameters[@"limit"] = limit
}

if (beforeUserID) {
...
etc
```